fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/AppBarTokens.kt

```diff
@@ -28,6 +29,11 @@ enum class AppBarSize {
     Small
 }
 
+enum class TitleAlignment {
```

Comment:
This can be removed.

Reply:
this is being used in the activity class